charts/metrics-server/templates/deployment.yaml

```diff
@@ -74,6 +78,11 @@ spec:
           volumeMounts:
             - name: tmp
               mountPath: /tmp
+            {{- if or .Values.tls.enable .Values.certManager.enabled }}
+            - mountPath: /var/run/serving-cert
+              name: volume-serving-cert
```

Comment:
Do you think something like `tls-cert` would be clearer?

Reply:
For the volume and volume mount names that is.

Reply:
Done